src/utils/constants/index.ts

```diff
@@ -1,2 +1,3 @@
 export const BANNER_DEFAULT_TIMEOUT = 3;
-export const a = '';
+// eslint-disable-next-line prefer-destructuring
+export const env = import.meta.env;
```

Comment:
```suggestion
export const getEnv = (env: string): string => import.meta[env] ?? '';
```

to use it you will now do
`getEnv('VITE_CRYPTOJS_KEY')`

this will eliminate the need to wrap env in String()

Please make this update every env is used and push to this za branch directly